assets/js/script.js

```diff
@@ -0,0 +1,24 @@
+function openNav() {
```

Comment:
[ OPTIONAL ] Please add all unnecessary files and folders like `.vscode` to `.gitignore`.